### PR TITLE
committed by Jishnu

### PR DESCRIPTION
**Pull request by JishnuGVarier**

### CODE REVIEW

File: tinkerers/JISHNU_G_VARIER.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Jishnu G Varier
+
+
+
+Hi there, I am Jishnu G Varier
+
+I am currently pursuing my BTech in Computer Science(Artificial Intelligence) at Muthoot Institute of Technology and Science, Varikoli, Ernakulam. I am passionate about the logical part involved in coding. I am looking forward to sharpening my skills and learning new skills. 
+
+## My Socials
+- [Linkedin] (http://www.linkedin.com/in/jishnu-g-varier-b3513a227)
```
Comment (aravinds-arv) [inline tinkerers/JISHNU_G_VARIER.md:10]: Heyy Jishnu, there shouldn't be a space between the square brackets and the rounded brackets. Please remove it to fix the hyperlink

**Review by aravinds-arv — CHANGES_REQUESTED** [review]

**Comment by JishnuGVarier** [comment]
done with changes

**Comment by aravinds-arv** [comment]
Merged 🎉

**Comment by JishnuGVarier** [comment]
Ok thank you

**Pull request merged**